### ISSUE TITLE
When switching to a new page, voiceover doesn't announce the new page title or header

### ISSUE THREAD

**Issue by smcmurtry** [veteransaffairscanada/vac-benefits-directory#1478]
This happens both with next/link and with `setSection` `onClick` events in the guided experience.

**Comment by smcmurtry** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [Accessibility: Screenreader Navigation](https://trello.com/c/knKtuM4T/100-accessibility-screenreader-navigation)

**Comment by brdunfield** [comment]
@smcmurtry was this closed by #1649?

**Comment by smcmurtry** [comment]
@brdunfield no that PR just solved the issue for /benefits-directory. The issue still exists for all the other pages.

**Comment by smcmurtry** [comment]
Now that the GE has been split into pages this should be easier to solve. I've noticed that clicking the back link (not browser back) has the correct behaviour where the page title is announced and focus goes to the top of the page.

**Comment by smcmurtry** [comment]
I've now tried putting autofocusing headers into the GE, but this doesn't seem to affect the voiceover focus ([which is different from the keyboard focus](https://webaim.org/discussion/mail_thread?thread=8526) btw). This is weird because I based the autofocusing off of what IRCC did and theirs does seem to affect the voiceover focus: https://vancouver.rescheduler-dev.cds-snc.ca/register

**Comment by JuliannaR** [comment]
@smcmurtry if your having troubles, something could be a bug with voiceover, can you try validating with NVDA? If not, send it my way and I can see if I can help.

**Comment by smcmurtry** [comment]
@JuliannaR If you could try to reproduce this issue with another screen reader that would be a huge help! I only have access to voiceover currently.
